docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc

```diff
@@ -9,102 +9,108 @@ The following chapter instructs you how to create business rules in {PRODUCT} us
 [float]
 == Prerequisites
 
-* Finished data object in {PRODUCT}. For further information, see <<_defining_a_data_model>>.
+Finished data object in {PRODUCT}. For further information, see <<_defining_a_data_model>>.
 
 == Creating Guided Rules
 
 To create a guided rule:
```

Comment:
I think starting this with a full sentence "Use the following procedure to create a guided rule:" would be better. I vaguely remember a discussion with a translator about this sort of sentence fragment being an issue for them. I'm guilty of applying this inconsistently myself, so feel free to ignore me.

Reply:
There are actually 2 intro lines like that in this topic. I simply removed them. The section title is really all that is needed. That's my version of minimalism :)